### PR TITLE
Migrate moment to dayjs for Dnn.PersonaBar.UI

### LINKED ISSUE #3875: Replace momentjs with dayjs
## Description of problem
We are currently using Moment.js in various areas of DNN.  

## Description of solution
We should evaluate replacing with `dayjs` (which is much more lightweight and has a similar API).

The following are all the areas that need to be updated.  We'll use this as a checklist for PRs (one PR per area).

### Admin Experience
* [x] Dnn.EditBar.UI #5082 
* [x] Dnn.PersonaBar.UI #5107 
* [x] Dnn.React.Common #4466 
* [x] AdminLogs.Web #4464 
* [x] Pages.Web #4463 
* [x] Prompt.Web #4465
* [x] Roles.Web #4462
* [x] Servers.Web #4104
* [x] SiteImportExport.Web #4461
* [x] Sites.Web #4459 
* [x] Users.Web #4108

### Platform
* [x] DotNetNuke.Web #5083
* [x] Website #5083

## Description of alternatives considered
n/a

## Screenshots
n/a

## Additional context
We could go from approximately 18.2 KB to 2 KB.

## Affected browser
* [x] Chrome
* [x] Firefox
* [x] Safari
* [x] Internet Explorer
* [x] Edge

### PR DESCRIPTION
**Pull request by david-poindexter**
## Summary
* Migrate `moment` to `dayjs` for `Dnn.PersonaBar.UI`.
* Remove unused functions from `util.js`.
* Remove dateTimePicker knockout components (`koComponents`).
* Remove `datePicker.js` from knockout binding handlers (`koBindingHandlers`).
* Update all `.eslintskipwords.js` to remove `pikaday`.
* Remove `pikaday` from `Dnn.PersonaBar.UI` because it was ultimately never used.  `react-day-picker` is used instead.
* Update referenced license info to remove `pikaday`.

Resolves #3875

### CODE REVIEW

**Review by valadas — APPROVED** [review]
This looks good to me, I can confirm that the removed things appear to not be used by anything and I downloaded this build and tried various date pickers in the persona bar modules, everything looks good to me.THanks!
🌮

File: Dnn.AdminExperience/ClientSide/Bundle.Web/package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "babel-polyfill": "6.26.0",
     "create-react-class": "^15.6.3",
     "css-loader": "2.1.1",
+    "dayjs": "^1.11.1",
```
Comment (bdukes) [inline Dnn.AdminExperience/ClientSide/Bundle.Web/package.json:27]: We're including this via npm and also in the `contrib` folder, do we need both of these?

Comment (valadas) [reply Dnn.AdminExperience/ClientSide/Bundle.Web/package.json:27]: Well, in this context yes, we could use the build to copy the file over but it is a bigger proposition.

Comment (bdukes) [reply Dnn.AdminExperience/ClientSide/Bundle.Web/package.json:27]: I just wanted to ensure we're using both approaches and not making copies needlessly. If they're both in use, that's fine (though de-duplicating would be a bonus)

**Review by bdukes — APPROVED** [review]

File: Dnn.AdminExperience/ClientSide/Bundle.Web/package.json (hunk 1 above)
Comment (david-poindexter) [reply Dnn.AdminExperience/ClientSide/Bundle.Web/package.json:27]: @bdukes I totally agree with you.  It is a real mess in here - mainly due to the use of `requirejs`.

**Pull request merged**